ert_gui/__init__.py

```diff
@@ -1,10 +1,7 @@
 import sys
 
-REQUIRED_VERSION_HEX = 0x02070000
-
-
-if sys.hexversion < REQUIRED_VERSION_HEX:
-    raise Exception("ERT GUI Python requires at least version 2.7 of Python")
+if sys.version_info < (3,6):
```

Comment:
Is this necessary? I don't know of any other project that checks the Python version to tell you to sod off.

Reply:
Agree, will remove it